### ISSUE TITLE
Feature Request: Improve Email Log

### ISSUE THREAD

**Issue by planetahuevo** [sebastianfeldmann/phpbu#266]
Hi,

TLDR:
Add an option to send the failed backups notification to a different address (or addresses).

I come back with a new suggestion about emails. 
So far improving logging to add other sources https://github.com/sebastianfeldmann/phpbu/issues/143 is more complicated. 
So I am using email for log so far.
I added one feature here https://github.com/sebastianfeldmann/phpbu/issues/183 and this seems to help with that https://github.com/sebastianfeldmann/phpbu/pull/234. 

This one is a different approach. At the moment the only options are right now the subject flags (which are really useful) and with the option "sendOnlyOnError". 
I was thinking that should be straight forward to add an option to send to a different  email address (or several email addresses) when the backup fails (for whatever reasons). 
This way we can have the log for all the success on the nice OK folder and we can notify any other people on the team when the backup fails, but keeping them on the ERROR log folder at the same time. 

What do you think?
